crates/uv-resolver/src/resolution/graph.rs

```diff
@@ -243,7 +245,27 @@ impl ResolutionGraph {
             overrides: overrides.clone(),
             options,
             fork_markers,
-        })
+        };
+        let mut conflicting = graph.find_conflicting_distributions();
+        if !conflicting.is_empty() {
+            tracing::warn!(
+                "found {} conflicting distributions in resolution",
+                conflicting.len()
+            );
+        }
+        // When testing, we materialize any conflicting distributions as an
+        // error to ensure any relevant tests fail. Otherwise, we just leave
+        // it at the warning message above. The reason for not returning an
+        // error "in production" is that an incorrect resolution may only be
+        // incorrect in certain marker environments, but fine in most others.
+        // Returning an error in that case would make `uv` unusable whenever
+        // the bug occurs, but letting it through means `uv` *could* still be
+        // usable.
```

Comment:
Are there cases when this is not a bug? Otherwise we should attach the bug reporting URL to the warning.

Reply:
I don't believe so. I've added `https://github.com/astral-sh/uv/issues/new` to the warning message. Good call. :)